package.json

```diff
@@ -40,5 +42,10 @@
     "exclude": [
       "test/*.js"
     ]
+  },
+  "standard": {
+    "ignore": [
+      "docs/"
+    ]
```

Comment:
I guess this was only needed for the generated API stuff and is not need anymore. Can we remove it?